packages/fhir-converter/src/templates/cda/Header.hbs

```diff
@@ -79,14 +79,16 @@
             {{/with}}           
         {{/if}}
     {{/if}}
-            
-    {{#if document.author.assignedAuthor}}
-        {{#with (evaluate 'Utils/GeneratePractitionerId.hbs' obj=document.author.assignedAuthor) as |practitionerId|}}
-            {{>Resources/Practitioner.hbs practitioner=document.author.assignedAuthor ID=practitionerId.Id}},
+
+    {{#each (toArray document.author) as |author|}}
+      {{#if author}}
```

Comment:
we removed the check here if assignedAuthor is defined. 
this will now create even if its undefined and result in us creating empty practitioners. Lets add it back to be if author.assignedAuthor